docs/csharp/language-reference/compiler-options/deterministic-compiler-option.md

```diff
@@ -21,7 +21,7 @@ Causes the compiler to produce an assembly whose byte-for-byte output is identic
 
 ## Remarks
 
-By default, compiler output from a given set of inputs is unique, since the compiler adds a timestamp and a GUID that is generated from random numbers. You use the `-deterministic` option to produce a *deterministic assembly*, one whose binary content is identical across compilations as long as the input remains the same.
+By default, compiler output from a given set of inputs is unique, since the compiler adds a timestamp and a MVID that is generated from random numbers. You use the `-deterministic` option to produce a *deterministic assembly*, one whose binary content is identical across compilations as long as the input remains the same. In such a build the timestamp and MVID fields will be replaced with values derived from a hash of all the compilation inputs.
```

Comment:
```suggestion
By default, compiler output from a given set of inputs is unique, since the compiler adds a timestamp and a MVID that is generated from random numbers. You use the `-deterministic` option to produce a *deterministic assembly*, one whose binary content is identical across compilations as long as the input remains the same. In such a build, the timestamp and MVID fields will be replaced with values derived from a hash of all the compilation inputs.
```